### DISCUSSION TITLE
Invalid ciphertext header

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [spatie/laravel-ciphersweet#68] [Q&A]
Hi,
i setup the package and encrypt one field and it works well, then i add other field to configure bland indexing and it show me Invalid ciphertext header, i know need to do encryption step again, but i am asking is there away to solve that when fail to decrypt just return the filed value.
Thanks.

**Comment by MikeCraig418** [comment]
@abdosaeedelhassan, I received this error today as well. In my model, I was querying using the select(...) statement for an individual field. My select() statement needed to have all encrypted fields. Then the error went away.

**Reply by abdosaeedelhassan** [reply]
i mean that some time after doing encrypting step if there field added again as plain text or there are old data not encrypted, this operation throw error i think a avoiding it by letting decrypt function to check if the text if plain text return it only otherwise doing decrypting.
